drivers/net/Kconfig

```diff
@@ -191,6 +191,13 @@ config NET_LOOPBACK_MTU
 	help
 	  This option sets the MTU for loopback interface.
 
+config NET_LOOPBACK_DROP_ALL_PACKETS
```

Comment:
There's already `config NET_LOOPBACK_SIMULATE_PACKET_DROP`, isn't this new config duplicating functionality? The application can set drop ratio to `1.0` with `loopback_set_packet_drop_ratio()`.

Reply:
True, but it also simulates real driver by adding clone, so, to avoid clone impacting the network stack bench mark for which this PR is used, added a new option.

Reply:
Moving that before clone, won't it change the current behaviour? Let me run some tests to see the performance impact, if its minimal then will use the existing define itself.

Reply:
Well, the cloned packet is just dropped after being cloned, so skipping the cloning should not break anything.

Reply:
Sure, but the big comment says this is to simulate the real drivers (probably adds latency)

Reply:
That comment is not related to the packet drop functionality, it was added when the wrong loopback behavior was fixed (previously the original packet was forwarded back to the RX path, instead of a clone: https://github.com/zephyrproject-rtos/zephyr/commit/0436ad9e78de4c5d7a3d820e95778523d9c1be78).

Reply:
Ah makes sense now, thanks. Will remove the new define.